### ISSUE TITLE
Can't initialize state entry with null value.

### ISSUE THREAD

**Issue by FIIFE** [caesuric/mountain-goap#7]
Hi,

I have a `Target` state that I would like to be empty initially.
Only when something enters agents vision the target should be populated.
So I tried to initiate it with `null` value but it results in the following error:
```
NullReferenceException: Object reference not set to an instance of an object
MountainGoap.ActionNode.StateMatches (MountainGoap.ActionNode otherNode) (at Assets/mountain-goap-0.6.5/MountainGoap/Internals/ActionNode.cs:82)
```

It's not that big of a deal. I had to resort to initiating the entry with an empty object like this:
`{ AIStates.Target, new PositionTracker() }`
And it works but it would be convinient if I had the ability to just pass `null`.

If anything the more serious problem might be that the error is not clear as to what is wrong, I had to dig up a little to figure out that it's about state not accepting `null`.

Thanks!

**Comment by caesuric** [comment]
This should be fixed now in version 0.6.7. Let me know if it works for you and I'll close the issue.

**Comment by FIIFE** [comment]
Awesome, it works! Thanks!
